src/Store/CookieStore.php

```diff
@@ -200,7 +200,11 @@ public function setState(): self
                     $_COOKIE[$cookieName] = $chunk;
 
                     // Push the updated cookie to the host device for persistence.
-                    setcookie($cookieName, $chunk, $setOptions);
+                    // @codeCoverageIgnoreStart
```

Comment:
Tests cannot currently run in an isolated process due to a PHPUnit bug, and by extension lack of support within Pest. As such, all our tests that deal with cookies throw warnings because headers will have already been sent by previous tests. Until our unit testing framework fixes process isolation, we'll just skip invoking the cookie issuing methods to avoid triggering these errors.